test/lib/cli-entry.js

```diff
@@ -161,3 +161,13 @@ t.test('unsupported node version', async t => {
     /npm v.* does not support Node\.js 12\.6\.0\./
   )
 })
+
+t.test('node prerelease is supported inside engines', async t => {
+  const { cli, logs } = await cliMock(t, {
+    globals: {
+      'process.version': `${process.version}-pre.0`,
+    },
+  })
+  await cli(process)
+  t.notMatch(logs.warn[0], [])
```

Comment:
This asserts works, but I think it would be clearer if written as:
```suggestion
  t.strictSame(logs.warn[0], [])
```